.gitlab/deploy_packages/nix.yml

```diff
@@ -219,26 +219,61 @@ deploy_staging_iot_agent:
     - $S3_CP_CMD ./agent $S3_DSD6_URI/linux/iot/agent-$PACKAGE_VERSION --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers full=id=3a6e02b08553fd157ae3fb918945dd1eaae5a1aa818940381ef07a430cf25732
 
 # Datadog Installer
+
+# Common jobs
+
+# The installer is a special case because it's built by the datadog-agent pipeline, but
+# is expected to be deployed in a dedicated folder, so we need to define a special base which
+# fetches packages from a custom value of S3_RELEASE_ARTIFACTS_URI as the default value
+# include datadog-agent as a product
+.deploy_installer_deb:
```

Comment:
Any reason not to move these templates to the deploy_common.yml file for consistency? Not a big deal, though.

Reply:
I hesitated but I guess it makes more sense, I'll update